examples/stress_tests/many_cubes.rs

```diff
@@ -439,7 +439,7 @@ const EPSILON: f64 = 0.36;
 fn fibonacci_spiral_on_sphere(golden_ratio: f64, i: usize, n: usize) -> DVec2 {
     DVec2::new(
         PI * 2. * (i as f64 / golden_ratio),
-        (1.0 - 2.0 * (i as f64 + EPSILON) / (n as f64 - 1.0 + 2.0 * EPSILON)).acos(),
+        acos((1.0 - 2.0 * (i as f64 + EPSILON) / (n as f64 - 1.0 + 2.0 * EPSILON)) as f32) as f64,
```

Comment:
The loss of precision here is not great, the method basically pretends to be an f64 methods while being an f32 one.

Do we have a double precision version somewhere?

Reply:
(Of acos I meant)

Reply:
> (Of acos I meant)

I looked and I didn't see one in bevy_math::ops or ops::f64.

Reply:
I now see acos_approx in glam

Reply:
f64 wasn't banned, so I should just put it back

Reply:
Given that it's example code, I don't think this will break anything down the line.